database/src/main/resources/db/migration/V1_142__create_change_history_table.sql

```diff
@@ -0,0 +1,23 @@
+-- New table to store route change history
+CREATE TABLE "detected-change-history" (
+  id SERIAL PRIMARY KEY,
+  "transport-service-id" INTEGER REFERENCES "transport-service" (id) ON DELETE CASCADE,
+  "route-hash-id" TEXT,
+  "change-type" "gtfs-route-change-type",
+  "change-key" TEXT,
+  "email-sent" TIMESTAMP,
+  "change-detected" DATE,
+  "different-week-date" DATE, -- Date when the change happens
```

Comment:
Food for thought: should a new table use a more describing name which would document itself?